boot/bootdata/hivesys.inf

```diff
@@ -1094,6 +1095,9 @@ HKLM,"SYSTEM\CurrentControlSet\Control\Nls\Language","1409",0x00000000,"l_intl.n
 HKLM,"SYSTEM\CurrentControlSet\Control\Nls\Language","1809",0x00000000,"l_intl.nls"
 HKLM,"SYSTEM\CurrentControlSet\Control\Nls\Language","1c09",0x00000000,"l_intl.nls"
 HKLM,"SYSTEM\CurrentControlSet\Control\Nls\Language","2009",0x00000000,"l_intl.nls"
+HKLM,"SYSTEM\CurrentControlSet\Control\Nls\Language","2409",0x00000000,"l_intl.nls" ; en-029
```

Comment:
'en-029 (Caribbean)' is "reserved".
https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-lcid/70feba9f-294e-491e-b6eb-56532684c37f
https://winprotocoldoc.z19.web.core.windows.net/MS-LCID/%5bMS-LCID%5d.pdf

Should I add it to hivecls.inf anyway? No, or commented out, or enabled?
https://github.com/reactos/reactos/blob/9f21cc3e98227e9dc2f8786e3091d350d8635454/boot/bootdata/hivecls.inf#L694-L704

Reply:
I think it's OK and it could be added in these lists.